tests/orderbook_sync.spec.js

```diff
@@ -147,18 +141,17 @@ suite('OrderbookSync', () => {
     });
   });
 
-  test('emits an error event on error (with AuthenticatedClient)', done => {
```

Comment:
Could one of these still use an authenticated client, to ensure we don't break backwards compatibility?

Reply:
Personally, I don't think we should have tests for that, because we're moving forward with the API. The benefit of previously changing the `AuthenticatedClient` properties and the backwards compatibility that comes with it is _**just a nice and elegant side effect**_ . Backwards compatibility guarantees can be good sometimes, but they can also hold things back long-term. 

Nonetheless, with that off my chest, **I'm happy to add an extra test with `AuthenticatedClient` if would make you feel more comfortable.** I'm just not sure this is the correct place to test that `AuthenticatedClient` quacks like an `auth` payload object of `{ key, secret, passphrase }`.

Reply:
An extra test using an instance of `AuthenticatedClient` was just added via 09be1e8.